tools/install.py

```diff
@@ -49,18 +50,25 @@ def run(target, user, config, toolchain, **kwargs):
         user = getpass.getuser()
 
     # Target location to install to
-    target_dir = "{0}@{1}:/home/{0}/".format(user, target)
+    if target != "local":
+        target_binary_dir = "{0}@{1}:/home/{0}/".format(user, target)
+        target_toolchain_dir = "{0}@{1}:/home/{0}/".format(user, target)
+    else:
+        target_binary_dir = b.project_dir + "/docker/nugus_sim/binaries"
+        target_toolchain_dir = b.project_dir + "/docker/nugus_sim/toolchain"
+
```

Comment:
It was already `rsync`ing to two different directories (on the robot) before, but because they need to be set to different paths if we are installing locally, it made sense to set them there